tests/test_amity.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from amity import Amity
+
+
+class AmityTest(unittest.TestCase):
+
+    def setUp(self):
+        self.building = Amity()
+
+    def test_populate_offices(self):
+        '''
+        Test whether the populate method
+        populates the building with office rooms
+        '''
+        self.building.populate()
```

Comment:
**Code Quality: Nitpick**

This is already called in the constructor. No need for this.